.rubocop.yml

```diff
@@ -295,3 +295,5 @@ Style/WhenThen:
 Style/ZeroLengthPredicate:
   Description: 'Use #empty? when testing for objects of length 0.'
   Enabled: true
+Style/NumericLiterals:
+  Enabled: false
```

Comment:
I guess it depend we actually prefer _ in a number literals?  Is that a standard ruby thing?  Seems strange to me but I'm new here.